### PR TITLE
Added fix for generation volume name 

### PR DESCRIPTION
**Pull request by vefimova**
Name is generated as `<container_name> + "_" + <ContainerPath with replaced "/" to "_">`

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/71)

### CODE REVIEW

**Comment by jellonek** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/71#-:-KTOYein9lSyWxJkfG1K:bnfp4nl)*

File: pkg/libvirttools/virtualization.go [hunk 1]
```diff
@@ -139,8 +140,9 @@ func (v *VirtualizationTool) processVolumes(mounts []*kubeapi.Mount, domXML stri
 	}
 
 	for _, mount := range mounts {
+		volumeName := containerName + "_" + strings.Replace(*mount.ContainerPath, "/", "_", -1)
```
Comment (nhlfr) [inline pkg/libvirttools/virtualization.go]: What if `mount.ContainerPath` is `nil`?

I'd suggest to do `mount.GetContainerPath()` - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/api/v1alpha1/runtime/api.pb.go#L386-L391

Comment (jellonek) [inline pkg/libvirttools/virtualization.go]: In this situation we should then verify if it's equal to `""`.
Is there any possibility if ContainerPath could be `nil`?
Is this really needed for POC?

Comment (nhlfr) [inline pkg/libvirttools/virtualization.go]: mount.ContainerPath is a pointer, so doing `*` when it's nil should just fail/panic

You may of course do manual checks, but these `GetSomething` methods in CRI are already doing that and returning empty structs or strings when pointers are null.

I know that we have a lot of places we are not using these methods, shame on me, I'm going to change them all soon.

Comment (vefimova) [inline pkg/libvirttools/virtualization.go]: fixed

**Pull request merged**